packages/jest-cli/src/cli/args.js

```diff
@@ -461,6 +461,11 @@ export const options = {
       'every single file.',
     type: 'boolean',
   },
+  runner: {
```

Comment:
Isn't this `testRunner`, `runner` is not a valid CLI arg:

![](http://dp.hanlon.io/3s3s2R2P3S00/Image%202018-02-28%20at%205.24.33%20PM.png)

Reply:
https://facebook.github.io/jest/docs/en/configuration.html#runner-string I see runner is under the section Options which is for package.json ?

Reply:
This is valid. The validation error is about missing module, not a "probably typo" in your config option

Reply:
Sorry, the screen is wrong, here it is with a valid runner: 

![](http://dp.hanlon.io/3B3S1I253P1e/Image%202018-02-28%20at%205.35.34%20PM.png)

Reply:
Weird that the order of op there is backwards

Reply:
This shouldn't happen on this branch, because the CLI validation goes through `args.js` – where `runner` is currently missing on master.

Reply:
Eslint asked me to put the op to be at that position. I found it weird too. Sorry having some troubles with internet connection now

Reply:
I'm a dufus, thanks @thymikee